skimage/transform/_hough_transform.pyx

```diff
@@ -314,7 +314,8 @@ def _hough_line(cnp.ndarray img,
 
 def _probabilistic_hough_line(cnp.ndarray img, int threshold,
                               int line_length, int line_gap,
-                              cnp.ndarray[ndim=1, dtype=cnp.double_t] theta):
+                              cnp.ndarray[ndim=1, dtype=cnp.double_t] theta,
+                              seed=None):
```

Comment:
@mcs07 clear, thanks! I think, the function could be made pure-Cython, but for now your solution is reasonable.